test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertFrom-Json.Tests.ps1

```diff
@@ -106,6 +106,17 @@ Describe 'ConvertFrom-Json Unit Tests' -tags "CI" {
         { $nestedJson | ConvertFrom-Json -AsHashtable:$AsHashtable } |
             Should -Throw -ErrorId "System.ArgumentException,Microsoft.PowerShell.Commands.ConvertFromJsonCommand"
     }
+
+    It 'Can convert null' {
+        'null' | ConvertFrom-Json | Should -Be $null
```

Comment:
Why do we send a string if we need a check for $null?

Reply:
The change is in `ConvertTo-Json`, but there should be the analog test for `ConvertFrom-Json` which is this.